ibexa/commerce/4.0.x-dev/encore/ez.webpack.config.manager.js

```diff
@@ -1,33 +1,44 @@
 const fs = require('fs');
-const findItems = (eZConfig, entryName) => {
-    const items = eZConfig.entry[entryName];
+const ibexifyEntryName = (entryName) => {
+    const ibexaEntryName = entryName.replace('ezplatform-', 'ibexa-').replace('ezcommerce-', 'ibexacommerce-');
```

Comment:
correct me if i'm wrong
```suggestion
    const ibexaEntryName = entryName.replace('ezplatform-', 'ibexa-').replace('ezcommerce-', 'ibexa-commerce-');
```